components/automate-dex/web/templates/login.html

```diff
@@ -2,36 +2,67 @@
 <!-- rendered using https://golang.org/pkg/html/template/-->
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <title>{{ issuer }}</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link href="{{ url .ReqPath "static/main.css" }}" rel="stylesheet">
-    <link href="{{ url .ReqPath "theme/styles.css" }}" rel="stylesheet">
-  </head>
-  <body>
-    <main class="theme-panel">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <title>{{ issuer }}</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href='{{ url .ReqPath "static/main.css" }}' rel="stylesheet">
+  <link href='{{ url .ReqPath "theme/styles.css" }}' rel="stylesheet">
+</head>
+
+<body>
+  {{ if eq ("showBanner" | extra) "true" }}
```

Comment:
In these blocks, we're checking that the user has set their instance to show the banner, if it does, then we go ahead and display the banner.  You'll see this similar type of block used both in this file and in `password.html`.  Checking against `extra` is a pattern that comes from DexIDP (https://dexidp.io/docs/templates/)